x-pack/plugins/siem/public/components/header_page/header_page.tsx

```diff
@@ -18,25 +18,41 @@ const Header = styled.header`
 `;
 
 export interface HeaderPageProps {
+  badgeLabel?: string;
+  badgeTooltip?: string;
   children?: React.ReactNode;
   subtitle?: string | React.ReactNode;
   title: string | React.ReactNode;
 }
 
-export const HeaderPage = pure<HeaderPageProps>(({ children, subtitle, title }) => (
-  <Header>
-    <EuiFlexGroup alignItems="center">
-      <EuiFlexItem>
-        <EuiTitle size="l">
-          <h1 data-test-subj="page_headline_title">{title}</h1>
-        </EuiTitle>
+export const HeaderPage = pure<HeaderPageProps>(
+  ({ badgeLabel, badgeTooltip, children, subtitle, title }) => (
+    <Header>
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem>
+          <EuiTitle size="l">
+            <h1 data-test-subj="page_headline_title">
+              {title}
+              {badgeLabel && (
+                <>
+                  {' '}
```

Comment:
you do not want just to add css

Reply:
@XavierM: Adding a CSS `margin-left` to the badge can be problematic in certain scenarios, such as line breaks (which could push the badge to a new line, but still have an undesired left margin). A simple space is more bulletproof and is just fine since it's an `inline-block` element. Plus, this appears to be how the Kibana design team uses it in their docs.